.github/workflows/run_mmperf.yml

```diff
@@ -54,7 +54,7 @@ jobs:
       - name: "Calculating version info"
         id: iree
         run: |
-          export IREE_SHA=$(git rev-parse --short HEAD)
+          IREE_SHA=`git rev-parse HEAD`
```

Comment:
You can already get this from the GitHub context: `github.sha` (see https://docs.github.com/en/actions/learn-github-actions/contexts). I think it's reasonable to alias it to `iree-sha` for clarity in the rest of the workflow if you want, but no need to export it from this step: the job outputs can just reference the github context and you can set the env var for this step using the context as well. Or if you don't want the alias, all the places where you use `needs.setup.outputs.iree-sha` could just use `github.sha`